Documentation/coding-guidelines/breaking-change-rules.md

```diff
@@ -158,16 +158,17 @@ Breaking Change Rules
 * Adding `virtual` to a member  
 > Make note, that marking a member virtual might cause previous consumers to still call the member non-virtually.
 
-* Introducing or removing an override
-> Make note, that introducing an override might cause previous consumers to skip over the override when calling `base`.
-
 &#10007; **Disallowed**
 * Adding an member to an interface
 
 * Adding an abstract member to a type when there _are accessible_ (`public` or `protected`) constructors and the type is not `sealed`
 
 * Adding a constructor to a class which previously had no constructor, without also adding the default constructor
 
+* Introducing or removing an override to non-sealed `protected` or `protected internal` types
```

Comment:
> > Should this refer to methods instead of types?

> All overridable members - methods and properties 

So should this say just "Introducing or removing an override in non-sealed types" instead?